client/pom.xml

```diff
@@ -155,10 +170,10 @@
                   <bindingFiles>
                     <bindingFile>bindings.xml</bindingFile>
                   </bindingFiles>
-                  <wsdlDirectory>${project.build.directory}/winrm4j-service-wsdl/META-INF/wsdl</wsdlDirectory>
+                  <wsdlDirectory>${basedir}/../service/target/classes/wsdl</wsdlDirectory>
```

Comment:
Would be nice to keep the `META-INF/wsdl` location as that's where the `wsgen` tool is looking at on the classpath.